src/components/ChatEntry.js

```diff
@@ -1,22 +1,34 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp'
+
 
 const ChatEntry = (props) => {
+  const changeLike = props.liked ? '❤️' : '🤍'; 
   return (
     <div className="chat-entry local">
-      <h2 className="entry-name">Replace with name of sender</h2>
+      <h2 className="entry-name">{props.sender}</h2>
       <section className="entry-bubble">
-        <p>Replace with body of ChatEntry</p>
-        <p className="entry-time">Replace with TimeStamp component</p>
-        <button className="like">🤍</button>
+        <p>{props.body}</p>
+        <p className="entry-time">
+          <TimeStamp time = {props.timeStamp}/></p>
+          <button className="like" onClick={() => {
+            props.updateLike(props.id)}
+          }>{changeLike}</button>
       </section>
     </div>
   );
 };
 
+
 ChatEntry.propTypes = {
```

Comment:
Yay, prop types! It would be good to make sure all of the props are here (like id).